lhotse/audio.py

```diff
@@ -231,8 +232,7 @@ def from_file(
         else:
             try:
                 # Try to parse the file using pysoundfile first.
-                import soundfile
-                info = soundfile.info(str(path))
+                info = sf.info(str(path))
```

Comment:
is there a missing case for using `sph_info`?

Reply:
Right, good catch.